dbt/models/qc/schema.yml

```diff
@@ -1,28 +1,52 @@
 models:
-  - name: qc.vw_class_mismatch
+  - name: qc.vw_oby_class_mismatch
```

Comment:
I refactored a bunch of these QC tests to make wider use of generics and simplify the configurations, since I needed to refactor them to select columns and add descriptions anyway.